recipe/meta.yaml

```diff
@@ -96,21 +104,17 @@ requirements:
     # no quesitons about the license of the resulting opencv
     # binary
     - ffmpeg {{ ffmpeg }}=lgpl_*
-    - qt-main                             # [not osx and not ppc64le]
+    - qt-main     # [qt_version == 5]
```

Comment:
Would `!= none` be better, considering our move to qt6 at some point?

Reply:
Honestly, my branch is called `qt6` because i tried to get qt6 to work. I just couldn't, so i kinda "gave up" and reverted to this simpler one.

Reply:
I see. All good then :)